src/models/message-trait.ts

```diff
@@ -1,3 +1,24 @@
 import type { BaseModel } from "./base";
+import type { CorrelationIdInterface } from "./correlation-id";
+import type { MessageExamplesInterface } from "./message-examples";
+import type { BindingsMixinInterface, DescriptionMixinInterface, ExtensionsMixinInterface, ExternalDocumentationMixinInterface, TagsMixinInterface } from './mixins';
+import type { SchemaInterface } from "./schema";
 
-export interface MessageTraitInterface extends BaseModel {}
+export interface MessageTraitInterface extends BaseModel, BindingsMixinInterface, DescriptionMixinInterface, ExtensionsMixinInterface, ExternalDocumentationMixinInterface, TagsMixinInterface {
```

Comment:
There are some missing methods from the Parser-API that I think should be implemented here (otherwise, discarded from the Parser-API).

- hasKnownSchemaFormat() : `boolean`
- schemaFormat() : `string`

See https://github.com/asyncapi/parser-api/blob/master/docs/v1.md#message

Reply:
I didn't add it on purpose for a simple reason. Defining `schemaFormat` at the messag level on a lot of problems from a schema reuse point of view, and I will try to move schemaFormat to the Schema object - then this will be removed. Here is my issue for 3.0.0 https://github.com/asyncapi/spec/issues/622. If we want it I can add it, but I hope that in 99% we will remove it.

Reply:
It makes sense indeed. How would you implement that? I think this should be then done in this PR

Reply:
I can add `schemaFormat` in this PR to `Message` model, but we can always remove it.

Reply:
Yeah, makes sense.

I like the `message.payload().schemaFormat()` you suggested in https://github.com/asyncapi/spec/issues/622. Maybe we can move forward with it later on, and now just adding an issue to work on it next?

Reply:
Issue :) https://github.com/asyncapi/parser-js/issues/544